src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 2px;
```

Comment:
The line-height is set to 2px, which is unusually small and may cause text to be cramped. Consider using a more typical line-height value, such as 1.5 or 150%, to improve readability.